final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +107,83 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+name: "Blarg",
+smell: "Sewage",
+weight: 7000,
+citiesDestroyed: ["Townsville", "Gotham", "Miami"],
+luckyNumbers: [4, 13, 6, 86],
+address: {"number" => 123, "street" => "sesame street", "State" => "New York",
```

Comment:
Generally you want to format your hashes with key/value pairs on separate lines, like so:

```
address: {
   "number" => 123,
   "street" => "sesame street",
   "State" => "New York",
   "Zip" => 12345
}
```

Reply:
I have corrected the hash format. Thank you!